js/src/views/Signer/components/TransactionPending/TransactionPending.js

```diff
@@ -65,20 +65,24 @@ export default class TransactionPending extends Component {
   render () {
     const { totalValue } = this.state;
     const className = this.props.className || '';
+
+    const { gasPriceEthmDisplay, gasToDisplay } = this.state;
+    const { id, date, data } = this.props;
+
     return (
       <div className={ `${styles.container} ${className}` }>
         <div className={ styles.mainContainer }>
           <TransactionMainDetails
             { ...this.props }
             className={ styles.transactionDetails }
             totalValue={ totalValue }>
-            <div className={ styles.iconsContainer }>
-              { this.renderGasPrice() }
-              { this.renderData() }
-            </div>
-            <div className={ styles.expandedContainer }>
-              { this.renderDataExpanded() }
-            </div>
+            <TransactionSecondaryDetails
```

Comment:
Is it really worth creating an etirely new component? I don't have strong opinions on this, but it feels bloated.